di-ipv-credential-issuer-stub/core-dev-deploy/face-to-face/template.yaml

```diff
@@ -479,7 +479,7 @@ Resources:
           - Name: F2F_STUB_QUEUE_API_KEY
             Value: !Ref F2fStubQueueApiKey
           - Name: F2F_STUB_QUEUE_NAME
-            Value: !If [IsCoreDev, !Sub "stubQueue_F2FQueue_${Environment}", "stubQueue_F2FQueue_build"]
+            Value: !If [IsCoreDev, !Sub "stubQueue_criResponseQueue_${Environment}", "stubQueue_criResponseQueue_build"]
```

Comment:
This will presumably break the queue functionality briefly in dev and build when it is deployed. Please make sure to warn Jet and Hilux to not merge anything while you're releasing this or their pipeline tests might fail.

Reply:
It should only update the placeholder value for the queue input. The tests may not rely on this default value, so there's a chance it'll pass but I'll do due diligence nevertheless.